Jamulus.pro

```diff
@@ -93,22 +93,31 @@ win32 {
 
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
-        message(Using Jack instead of ASIO.)
-
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "i386") {
+            message("Native i386 build")
+            libjackname = libjack.lib
+        } else {
+            message("Native x86_64 build")
+            libjackname = libjack64.lib
+            !exists("C:/Program Files (86)/JACK2/include/jack/jack.h") {
```

Comment:
"C:/Program Files (86)/JACK2/include/jack/jack.h" is missing an x, it should be

C:/Program Files (**x**86)/JACK2/include/jack/jack.h

Reply:
Ok in 02db193